components/chainhook-cli/src/service/tests/helpers/mock_stacks_node.rs

```diff
@@ -8,7 +8,7 @@ use chainhook_sdk::types::{
     STXTransferEventData, SmartContractEventData, StacksTransactionEventPayload,
```

Comment:
This file's changes are to allow adding a `fork_id` and `parent_fork_id` to the stacks blocks that are mined for tests.